src/posts/announce-2.0.0.Final.adoc

```diff
@@ -0,0 +1,168 @@
+---
+title: Announcing Thorntail 2.0.0.Final
+publishDate: 2018-06-27
+modifyDate: 2018-06-27
+author: The Thornail Team
+layout: post.jade
+---
+
+== Thorntail 2.0.0.Final
+
+The new name of WildFly Swarm is _Thorntail_! Enjoy the freedom of flying high like a Thorntail bird!
+
+=== What's New?
+
+* Did we say we have a new name: Thorntail?
+* We give up the `YYYY.MM.VV` versioning scheme and switched to a more convenient `major.minor.micro` versioning
```

Comment:
If "switched to", then should be "gave up"?

Reply:
What about "We have left"...?